package.json

```diff
@@ -82,8 +82,7 @@
     "zip": "zip -ur bridge-$npm_package_version.zip README.md bridge-https.py checksums.txt && cd build && zip -ur ../bridge-$npm_package_version.zip ./* && cd ..",
     "release": "npm install && npm run build && npm run checksum && npm run zip",
     "pilot:ganache": "ganache-cli --blockTime 1 --networkId 1 --host '0.0.0.0' -m 'benefit crew supreme gesture quantum web media hazard theory mercy wing kitten' > /dev/null &",
-    "pilot:deploy": "cd node_modules/azimuth-solidity && truffle deploy",
-    "pilot:setup": "npm run pilot:ganache && npm run pilot:deploy",
+    "pilot:setup": "npm run pilot:ganache && truffle deploy",
```

Comment:
Maybe we want `pilot-with-invites` etc shorthands here? Always feel explicit commands are nicer than manually filling in environment variables.